2017/JULY/JULY17/EXPTREE/EXPTREE.py

```diff
@@ -0,0 +1,52 @@
+#__BY:sirjan13__
+
+def gcd(n,m):
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpufcxdz6g/2017/JULY/JULY17/EXPTREE/EXPTREE.py
+++ b/tmp/tmpufcxdz6g/2017/JULY/JULY17/EXPTREE/EXPTREE.py
@@ -1,11 +1,14 @@
 #__BY:sirjan13__
 
-def gcd(n,m):
-    if n==0:
+
+def gcd(n, m):
+    if n == 0:
         return m
-    return gcd(m%n,n)
+    return gcd(m % n, n)
 
-t1,t2=0,0
+
+t1, t2 = 0, 0
+
 
 def egcd(a, b):
     x,y, u,v = 0,1, 1,0
```